colorizer_data/converter.py

```diff
@@ -330,18 +411,25 @@ def convert_colorizer_data(
             image. Defaults to "Centroid X."
         centroid_y_column (str): The name of the column containing y-coordinates of object
             centroids, in pixels relative to the frame image, where 0 is the top edge of the image.
-            Defaults to "Centroid Y.""
+            Defaults to "Centroid Y."
         outlier_column (str): The name of the column containing outlier flags. 0/False indicates a
             normal object, while 1/True indicates an outlier. Outliers are excluded from min/max
             calculation for features. Defaults to "Outlier."
-        backdrop_columns (List[str] | None): A list of column names containing file paths to
-            backdrop images. If set, these images will be copied and included in the dataset as
-            backdrops that can be toggled. Defaults to `None`.
-        backdrop_info (Dict[str, BackdropMetadata] | None): A dictionary mapping column names to
-            `BackdropMetadata` metadata. This includes the backdrop's name, description, and
-            file paths. If the files do not exist in the dataset directory, the files
-            will be copied to it and the paths updated to be relative to the manifest. Defaults to
-            `None`.
+        backdrop_column_names (List[str] | None): A list of column names containing file paths to
+            backdrop images. If the images are not already inside the output directory,
+            they will be copied into it. Defaults to `None`.
+        backdrop_info (Dict[str, BackdropMetadata] | None): A dictionary mapping backdrop names to
+            `BackdropMetadata` overrides. This includes the backdrop's name, key, and optionally
+            relative paths to the frames. Defaults to `None`.
+            - If the name matches a backdrop column name in `data`, any defined fields will
+            override the default metadata for that column (e.g. `key` or `name`). If the `frame`
+            field is provided, those paths will be used instead of the original column values
+            when fetching or copying files. Frame paths will be edited to be relative to the
+            dataset directory, and copied into the directory if they are not already subfiles.
+            - If the name does not match a backdrop column name, a new backdrop will be directly
+            added with the provided metadata. Frame paths will NOT be edited, and no files will
+            be copied. (This should only be done if the files are already in the output directory,
+            and the paths are already relative to the directory.)
```

Comment:
Really tried to be specific about what these did and they ended up MUCH more detailed than their neighbors. Let me know if you see ways to improve this!

Reply:
I can also see a world in which I as a scientist would want to provide paths relative to wherever the current script I'm running is and have the converter API copy those into the directory for me. I'm not sure what the best way to specify that copying behavior should be.